internal/releases/repo_github.go

```diff
@@ -41,41 +41,95 @@ func (r *githubRepository) Process() error {
 	ctx := context.Background()
 	opts := &github.ListOptions{PerPage: 3}
 
+	// Check if the repository is archived
+	repoObject, _, err := client.Repositories.Get(ctx, r.org, r.info.Name)
+	if err != nil {
+		return fmt.Errorf(
+			"error while checking archived status for repo: %s/%s/%s: %v", r.org, r.team, r.info.Name, err,
+		)
+	}
+	r.info.IsArchived = repoObject.GetArchived()
+
+	// Get the releases from the repo
 	releases, _, err := client.Repositories.ListReleases(ctx, r.org, r.info.Name, opts)
 	if err != nil {
 		return fmt.Errorf(
 			"error listing releases for repo: %s/%s/%s: %v", r.org, r.team, r.info.Name, err,
 		)
-	} else if len(releases) == 0 {
-		return nil
-	}
+	} else if len(releases) != 0 { // If there are releases, add them
```

Comment:
I found this comment more distracting than just grokking the code ;-)

```suggestion
	} else if len(releases) != 0 {
```